data/json/overmap/overmap_terrain/overmap_terrain_nether_monster_corpse.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "type": "overmap_terrain",
+    "id": "corpse_bowels_empty_edge",
+    "name": "monster corpse bowels",
+    "sym": "O",
+    "color": "light_red",
+    "see_cost": 5,
+    "flags": [ "RISK_HIGH" ]
```

Comment:
While these are a work in progress you'll want to add the `SHOULD_NOT_SPAWN` flag to these OMTs so that they don't cause the `overmap_terrain_coverage` test to fail.